src/Data/Quantity/Physics.purs

```diff
@@ -33,19 +33,19 @@ speedOfLight = 299792458.0 .* meter ./ second
 
 -- | The Newtonian constant of gravitation.
 gravitationalConstant ∷ Quantity
-gravitationalConstant = 6.67408e-11 .* meter .^ 3.0 ./ (kilo gram <> second .^ 2.0)
+gravitationalConstant = 6.67408e-11 .* meter .^ 3.0 ./ (kilogram <> second .^ 2.0)
 
 -- | The Planck constant.
 planckConstant ∷ Quantity
 planckConstant = 6.626070040e-34 .* (joule <> second)
 
 -- | The reduced Planck constant.
 ℏ ∷ Quantity
-ℏ = planckConstant ⊘ (scalar 2.0 ⊗ pi)
+ℏ = planckConstant ⊘ tau
```

Comment:
I would personally keep this as 2 pi. ℏ =h/2pi is well known, but tau is not. IMO.

Reply:
Okay, I'll revert this change.